app/modules/asset_groups/resources.py

```diff
@@ -480,7 +480,7 @@ class AssetGroupSightingCommit(Resource):
     )
     # NOTE this returns a sighting schema not an AssetGroup one as the output of this is that
     # a sighting is created
-    @api.response(BaseSightingSchema())
+    @api.response(DetailedSightingSchema())
```

Comment:
Previouusly the response just contained the sighting guid. This seemed to be a far more sensible option (and made the tests easier too)